executors/sshexec/volume.go

```diff
@@ -228,3 +228,101 @@ func (s *SshExecutor) checkForSnapshots(host, volume string) error {
 
 	return nil
 }
+
+func (s *SshExecutor) VolumeInfo(host string, volume string) (*executors.SingleVolumeInfo, error) {
```

Comment:
Could this possibly be added in a separate patch, in order to keep the patches a little smaller?